x-pack/plugins/apm/public/hooks/useFetcher.test.tsx

```diff
@@ -4,17 +4,23 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { renderHook } from '@testing-library/react-hooks';
+import { renderHook, RenderHookResult } from '@testing-library/react-hooks';
 import { delay } from '../utils/testHelpers';
-import { useFetcher } from './useFetcher';
+import { FetcherResult, useFetcher } from './useFetcher';
 import { MockApmPluginContextWrapper } from '../context/ApmPluginContext/MockApmPluginContext';
+import { ApmPluginContextValue } from '../context/ApmPluginContext';
 
 // Wrap the hook with a provider so it can useApmPluginContext
 const wrapper = MockApmPluginContextWrapper;
 
 describe('useFetcher', () => {
   describe('when resolving after 500ms', () => {
-    let hook: ReturnType<typeof renderHook>;
```

Comment:
Something around types of `renderHook` has changed that it is now more strict and doesn't allow use `hook` with this inferred type